draft-jholland-quic-multicast.md

```diff
@@ -737,6 +737,14 @@ A client might receive multicast packets that it can not associate with any chan
 This traffic is presumed either to have been corrupted in transit or to have been sent by someone other than the legitimate sender of traffic for the channel, possibly by an attacker or a misconfigured sender.
 If these packets are addressed to an (S,G) that is used for reception in one or more known channels, the client MAY leave these channels with reason "Excessive Spurious traffic".
 
+## Retransmission of information
+In addition to the mechanisms used for retransmission described in {{Section 13.3 of RFC9000}} and {{Section 5.2 of RFC9221}} the following rules apply to the newly introduced frames:
+
+- As the properties carried in MC_ANNOUNCE frames can not change during the lifetime of a channel, information contained in them can be retransmitted one to one.
```

Comment:
What is "retransmitted one to one"?  Maybe we should phrase this like the same way the NEW_TOKEN handling is phrased, I think it's similar in that we just want to ensure the whole thing is properly received and retransmit it exactly the same again if we don't see an ack for it.